shelley/chain-and-ledger/formal-spec/protocol-parameters.tex

```diff
@@ -166,7 +166,7 @@ \section{Protocol Parameters}
     \begin{array}{r@{~\in~}lr}
       \SlotsPerEpoch & \N & \text{slots per epoch} \\
       \SlotsPerKESPeriod & \N & \text{slots per KES period} \\
-      \SlotsPrior & \Duration & \tau\text{ in \cite{ouroboros_praos}}\\
+      \SlotsPrior & \Duration & \text{nonce leakage parameter }\tau\text{ in \cite{ouroboros_praos}}\\
```

Comment:
thank you for clearing this us @nfrisby ! I had not realized that there were two `\tau`s, and "nonce leakage parameter" is better anyway.